reactivesocket-core/src/main/java/io/reactivesocket/Plugins.java

```diff
@@ -24,4 +25,30 @@
 
     private Plugins() {}
 
+    static {
+        if (Boolean.getBoolean("io.reactivesocket.fragmentation.enable")) {
+            int mtu = Integer.getInteger("io.reactivesocket.fragmentation.mtu", 1024);
+
+            if (Plugins.DUPLEX_CONNECTION_INTERCEPTOR == null) {
```

Comment:
What is the reasoning behind the system property plugin mechanism?  Shouldn't this be driven by individual applications wiring up connections as they see fit?  Could this be expressed non-statically via a helper that wraps a DuplexConnection with sensible defaults?